### ISSUE TITLE
container cpu usage some too high, some zero.

### ISSUE THREAD

**Issue by minesunny** [containerd/nerdctl#2569]
### Description

container cpu usage some too high, some zero.

### Steps to reproduce the issue

1.
2.
3.

### Describe the results you received and expected

nerdctl stats --namespace=k8s.io

https://github.com/containerd/nerdctl/assets/98507458/2d56b704-cfdb-40c1-9d27-0dc2448c709b

crictl stats
![image](https://github.com/containerd/nerdctl/assets/98507458/1be0e8bf-5aaf-40a6-ac2c-bcfdc04d74f2)

### What version of nerdctl are you using?

1.6.2

### Are you using a variant of nerdctl? (e.g., Rancher Desktop)

None

### Host information

<img width="699" alt="image" src="https://github.com/containerd/nerdctl/assets/98507458/19a3d865-a834-47df-83bb-a94a4885cac8">
![image](https://github.com/containerd/nerdctl/assets/98507458/f914dcbd-54fe-477d-9c05-c2f16825f073)

**Comment by yankay** [comment]
HI @minesunny 

When try to invesgate the issue with [stress-ng](https://github.com/alexei-led/stress-ng), the result is no zero :-)

https://github.com/containerd/nerdctl/assets/1469319/513cef21-9ec6-4f75-abd9-cd83906db382

the nerdctl version:
```
nerdctl --version
nerdctl version 1.6.0
```

Would you please share us how to reproduce the issue.

**Comment by minesunny** [comment]
> HI @minesunny
> 
> When try to invesgate the issue with [stress-ng](https://github.com/alexei-led/stress-ng), the result is no zero :-)
> 
>  2023-10-17.104716.mp4 
> the nerdctl version:
> 
> ```
> nerdctl --version
> nerdctl version 1.6.0
> ```
> 
> Would you please share us how to reproduce the issue.

I don't have exactly steps to reproduce the issue. I just install k8s cluster(master1,node1,node2)and deploy my apps,then I want to find a better ctr command line tools to help me manager my containers.Finally, this issue happened

**Comment by fahedouch** [comment]
@yankay did you stressed kubernetes containers ? Seems like issue occurs with k8s containers

**Comment by minesunny** [comment]
> @yankay did you stressed kubernetes containers ? Seems like issue occurs with k8s containers

No,so it is weird that cpu's usage up to hundreds percentage or ten  hundreds percentage;

**Comment by venkatasandeeplade** [comment]
I am also facing the same issue. @fahedouch  can you help us here ?

```shell
ineedshell@cop-fips-fresh-install-n1:/tmp$ sudo nerdctl --version
nerdctl version 1.4.0
ineedshell@fips-fresh-install-n1:/tmp$ sudo nerdctl --info
FATA[0000] unknown flag: --info
ineedshell@fips-fresh-install-n1:/tmp$ sudo nerdctl info
Client:
 Namespace:	default
 Debug Mode:	false

Server:
 Server Version: 1.6.33
 Storage Driver: overlayfs
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Log: fluentd journald json-file syslog
  Storage: aufs native overlayfs
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.4.0-1099-fips
 Operating System: Ubuntu 20.04.6 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 12
 Total Memory: 503.5GiB
 Name: cop-fips-fresh-install-n1.arubacorp.net
 ID: 54a9ebcb-74c3-4954-b920-87741ae88756

WARNING: No swap limit support
```
